aws/alarms/lambda/notify_slack/package.json

```diff
@@ -1,15 +1,23 @@
 {
   "name": "notify_slack",
   "version": "1.0.0",
-  "description": "Lambda that notifies multiple channels",
-  "main": "notify_slack.js",
+  "description": "Lambda that notifies Slack and triggers Opsgenie alert",
+  "main": "notify_slack.ts",
```

Comment:
The main file will still be `notify_slack.js` once the Typescript build folder is generated

Reply:
I'm having this problem where Jest can't import TypeScript files without a special flag. But, that flag only works with 'noEmit=true', which means no transpiled version is generated. It's really bugging me, this Jest and TypeScript combo. Thinking of ditching TypeScript for Jest and finding another solution.